binary-oracle-pair/README.md

```diff
@@ -0,0 +1,12 @@
+Simple Oracle Pair Token
+
+1. pick a deposit token
+2. pick the decider's pubkey
+3. pick the mint term end slot
```

Comment:
nit: Do we really need to stop minting at some point? Seems like this is better treated as a "decide term start slot" to indicate how long until a decision could be made. Even then, might not need a start slot.